app/workers/data_exporter.rb

```diff
@@ -28,6 +28,8 @@ def perform(importer_class, data_export_id)
     )
 
     Rails.logger.info 'Finished writing CSV. Sidekiq done'
+  ensure
+    RequestLocals.clear_all!
```

Comment:
Doesn't RequestStore::Sidekiq already do this? https://github.com/madebylotus/request_store-sidekiq/blob/master/lib/request_store/sidekiq/server_middleware.rb

It looks out of date with the guidance in https://github.com/ElMassimo/request_store_rails#usage-in-sidekiq